src/openrct2-ui/WindowManager.cpp

```diff
@@ -646,6 +646,127 @@ class WindowManager final : public IWindowManager
         }
         return nullptr;
     }
+
+    /**
+     * Finds the first window with the specified window class.
+     *  rct2: 0x006EA8A0
+     * @param cls (cl) with bit 15 set
+     * @returns the window or NULL if no window was found.
```

Comment:
```suggestion
     * @param WindowClass enum
     * @returns the window or nullptr if no window was found.
```
We've been slowly dropping the hex annotations, does it make sense to remove it here? Also the @ annotations seem outdated. This also applies to the other methods

Reply:
Agreed, no point in having documentation if it isn't updated. I'll see to it.

Reply:
I think Tulio means update the @ s, not remove them.

Reply:
Yes, that's what I was going to do as well. No worries.